nifi-nar-bundles/nifi-graph-bundle/nifi-other-graph-services/pom.xml

```diff
@@ -45,7 +46,7 @@
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-utils</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
+            <version>${project.version}</version>
```

Comment:
I do not recommend we change from 2.0.0-SNAPSHOT to ${project.version}.  In my experience this breaks during the release process.  The explicit versions get automatically handled.